lib/pages/editor-page.js

```diff
@@ -104,7 +104,7 @@ export default class EditorPage extends BaseContainer {
 
 	dismissMediaModal() {
 		const driver = this.driver;
-		const cancelSelector = webdriver.By.xpath( "//div[contains(@class, 'editor-media-modal')]//span[contains(@class, 'dialog__button-label') and contains(text(), 'Cancel' )]" );
+		const cancelSelector = webdriver.By.css( 'div.dialog__action-buttons > button:nth-child(1) > span' );
```

Comment:
This works, and is better than the selector which requires on English, but the `nth-child(1)` seems a bit hacky as it's really just the first child, so this would also work (`div.dialog__action-buttons button`) which is the first button.

What do you think about adding a data-e2e- attribute to this button?